hygraph-content-source/src/hygraph-content-source.ts

```diff
@@ -381,6 +382,17 @@ export class HygraphContentSource
             throw new Error('Error finding uploaded asset');
         }
 
+        let retry = 5;
+        let status;
+
+        do {
+            status = await this.client.getAssetUploadStatusById(assetId);
+            if (status === 'ASSET_CREATE_PENDING') {
+                await new Promise((resolve) => setTimeout(resolve, 1000));
+                retry--;
+            }
+        } while (status === 'ASSET_CREATE_PENDING' && retry >= 1);
```

Comment:
@arsenyg don't they have some kind of a webhook that tells when assets have been uploaded?
timeouts should always be last resort.